features/welcome.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Set up of the till
+  As a coffee shop employee
```

Comment:
I can see the utility in a feature like this but the fact you've called the feature "welcome" is a bit of a code smell - do you really need this feature? 

I tend to just have in my cucumber the core business logic that I require for my business to function. For example "welcome the coffee connection" is more a test of the design of the homepage, as opposed to testing that form inputs exist which would be a test of interaction which is what we are chiefly concerned with.